net_framework/DeepSpeechWPF/App.xaml.cs

```diff
@@ -20,7 +20,7 @@ protected override void OnStartup(StartupEventArgs e)
             {
                 //Register instance of DeepSpeech
                 DeepSpeechClient.DeepSpeech deepSpeechClient =
-                    new DeepSpeechClient.DeepSpeech("output_graph.pbmm");
```

Comment:
Maybe an extra comment here to put the link of the release page, like more model available...and put the link to releases.